docs/components/StyleGuideRenderer.tsx

```diff
@@ -42,33 +42,34 @@ function useTheme() {
   CSS animations should be disabled to ensure that animations don't 
   affect `react-styleguidist-visual` screenshots
 */
-if (window.location.port === '6061') {
-  injectGlobal`
+const GlobalStyle = createGlobalStyle`
   *, :before, :after {
     transition-property: none !important;
     transform: none !important;
     animation: none !important;
     font-family: sans-serif;
   }
 `;
-}
 
 const StyleGuideRenderer: React.SFC<StyleGuideRendererProps> = ({ children, ...props }) => {
   const theme = useTheme();
 
   return (
-    <ThemeProvider theme={theme}>
-      <Responsive
-        render={size => {
-          const Layout = size !== 'small' ? DesktopLayout : MobileLayout;
-          return (
-            <Layout logo={<Logo />} ribbon={<Ribbon />} {...props}>
-              {!window.location.hash ? <HomePage /> : children}
-            </Layout>
-          );
-        }}
-      />
-    </ThemeProvider>
+    <>
+      {window.location.port === '6061' && <GlobalStyle />}
```

Comment:
Woot :) Haven't seen this one. Why do we need this rule? What am I missing?

Reply:
do disable animations (for visual testing)

Reply:
Okay thanks good to know (alternatively could have been made work with an environment variable used when building for visual testing or a query parameter with a name like `visual_testing`)!

Reply:
may change with the next PR, I have some improvements for visual testing